tests/Builder/Query/Pipelines.php

```diff
@@ -67,6 +67,82 @@ enum Pipelines: string
     ]
     JSON;
 
+    /**
+     * AND Queries With Multiple Expressions Specifying the Same Field
+     *
+     * @see https://www.mongodb.com/docs/manual/reference/operator/query/and/#and-queries-with-multiple-expressions-specifying-the-same-field
+     */
+    case AndANDQueriesWithMultipleExpressionsSpecifyingTheSameField = <<<'JSON'
+    [
+        {
+            "$match": {
+                "$and": [
+                    {
+                        "price": {
+                            "$ne": {
+                                "$numberDouble": "1.9899999999999999911"
```

Comment:
We could also deviate from the suggested value and choose a value that doesn't have this problem and can be represented exactly (e.g. `0.2`).

Reply:
This is not an issue, as the value is always encoded the same way. But it can make tests to fail if we fix this behavior in a future version of the extension.

Tracking in https://jira.mongodb.org/browse/PHPC-2341
Other example: https://github.com/mongodb/mongo-php-builder/pull/22#discussion_r1450226432